hack/local-down-karmada.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+func() {
+    echo "Usage:"
+    echo "  hack/local-down-karmada.sh [-r]"
+    echo "  Description:"
+    echo "    r, remove-images, the local images needs to be removed"
+    exit -1
+}
+
+remove_images="false"
+while getopts 'r' OPT; do
+    case ${OPT} in
+        r) remove_images="true";;
+        ?) func;;
+    esac
+done
+
+#step1 remove kind clusters
+echo -e "\nStart removing kind cluster"
+kind delete cluster --name "${HOST_CLUSTER_NAME:-"karmada-host"}"
+kind delete cluster --name "${MEMBER_CLUSTER_1_NAME:-"member1"}"
+kind delete cluster --name "${MEMBER_CLUSTER_1_NAME:-"member2"}"
+kind delete cluster --name "${MEMBER_CLUSTER_1_NAME:-"member3"}"
+echo "Remove kind clusters successfully."
+
+#step2. remove kubeconfig
+echo -e "\nStart removing kubeconfig"
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+MAIN_KUBECONFIG=${MAIN_KUBECONFIG:-"${KUBECONFIG_PATH}/karmada.config"}
+MEMBER_CLUSTER_KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG:-"${KUBECONFIG_PATH}/members.config"}
+if [ -f ${MAIN_KUBECONFIG} ] ; then
```

Comment:
```suggestion
if [ -f "${MAIN_KUBECONFIG}" ] ; then
```

Double quote to prevent globbing and word splitting. The same problems below.